consensus/vbft/chain_store.go

```diff
@@ -133,15 +133,20 @@ func (self *ChainStore) AddBlock(block *Block) error {
 		return fmt.Errorf("chainstore AddBlock GetBlockExecResult: %s", err)
 	}
 	self.pendingBlocks[blkNum] = &PendingBlock{block: block, execResult: &execResult, hasSubmitted: false}
-	self.pid.Tell(
-		&message.BlockConsensusComplete{
-			Block: block.Block,
-		})
+	if _, present := self.pendingBlocks[blkNum-2]; present {
```

Comment:
If need clean here, loop all pendings to clean unused entries

Reply:
recover this delete pendingblock entry  to sumit function.